syntax = "proto3";

message RequestMessage
{
    int32 cmdType = 1;
    bytes clientId = 2;
    bytes serverId = 3;
    bytes data = 4;
    bytes sign = 5;
}

message ResponseMessage
{
    bool status = 1;
    bytes secretKeyId = 2;
    bytes clientId = 3;
    bytes serverId = 4;
    bytes data = 5;
};